index.html

```diff
@@ -1,197 +1,274 @@
-        <a class="toolbar-title" href="#"> Welcome </a>
-        <nav class="navbar">
-          <a href="#"> <img src="assets/images/hamburger.svg" alt="" /> </a>
-        </nav>
-      </header>
+  <title>Yousef Hesham</title>
+</head>
 
-      <section class="headline">
-        <h2 class="welcome-message">
-          Hey there.I’m Yousef <br />
-          <span>I’m a software developer </span>
-        </h2>
-        <p class="description">
-          I can help you build a product , feature or website Look through some
-          of my work and experience! If you like what you see and have a project
-          you need coded, don’t hestiate to contact me.
-        </p>
-        <div class="social">
-          <ul class="social-icons">
-            <li class="items">
-              <a href="#"> <img src="assets/images/twitter.svg" alt="" /></a>
+<body>
+  <div class="wrapper">
+
+    <header class="head-toolbar">
+      <a class="toolbar-title mobile-version"  href="#"> Welcome </a>
+      <nav class="navbar">
+        <a href="#" class="mobile-version"> <img src="assets/images/hamburger.svg" alt="" /> </a>
+        <ul class="nav-list">
+          <li class="nav-item"><a href="#works"> Portfolio</a></li>
+          <li class="nav-item"><a href="#about"> About</a></li>
+          <li class="nav-item"><a href="#contact"> Contact</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <section class="headline">
+      <h2 class="welcome-message">
+        Hey there.I’m Yousef <br />
+        <span>I’m a software developer </span>
+      </h2>
+      <p class="description">
+        I can help you build a product , feature or website Look through some
+        of my work and experience! If you like what you see and have a project
+        you need coded,<br> don’t hestiate to contact me.
+      </p>
+      <div class="social">
+        <ul class="social-icons">
+          <li class="items">
+            <a href="#"> <img src="assets/images/twitter.svg" alt="" /></a>
+          </li>
+          <li class="items">
+            <a href="#">
+              <img src="assets/images/Linkedinicon.svg" alt="" />
+            </a>
+          </li>
+          <li class="items">
+            <a href="#"> <img src="assets/images/medium.svg" alt="" /> </a>
+          </li>
+          <li class="items">
+            <a href="#"> <img src="assets/images/github.svg" alt="" /> </a>
+          </li>
+          <li class="items">
+            <a href="#"> <img src="assets/images/angellist.svg" alt="" /> </a>
+          </li>
+        </ul>
+
+      </div>
+      <img class="disabled-button" src="assets/images/Disabled.svg" alt="" />
+
+    </section>
+   </div> 
+
+    <section class="recent-works" id="works">
+      <h2 class="section-title">My Recent Works</h2>
+      <hr class="horizontal-line" />
+      <article class="card">
+        <div class="project-sample">
+          <h4 class="project-title">Multi-Post Stories Gain+Glory</h4>
```

Comment:
- [ ] Overall you have only 5 cards, but based on requirements the heading is followed by 2 rows with 3 projects each (6 projects in total). Please add one more cards ☀️
<img width="684" alt="Screenshot 2022-03-24 at 12 14 00" src="https://user-images.githubusercontent.com/77038610/159871678-807358ea-7648-426e-881e-7ecd81f4cc93.png">